zerver/openapi/markdown_extension.py

```diff
@@ -407,6 +409,11 @@ def extendMarkdown(self, md: markdown.Markdown) -> None:
         md.preprocessors.register(
             APITitlePreprocessor(md, self.getConfigs()), "generate_api_title", 531
         )
+        md.preprocessors.register(
+            ResponseDescriptionPreprocessor(md, self.getConfigs()),
+            "generate_response_description",
+            531,
```

Comment:
@MSurfer20, any particular reason to choose `531` as the priority number? As I can see that priority number 531 is already used by `APITitlePreprocessor`. And AFAIK, we keep these priority numbers distinct.
@timabbott can confirm this.